util/request/helper.go

```diff
@@ -55,11 +56,21 @@ func (r *Helper) GetBody(url string) ([]byte, error) {
 // decodeJSON reads HTTP response and decodes JSON body if error is nil
 func decodeJSON(resp *http.Response, res interface{}) error {
 	if err := ResponseError(resp); err != nil {
-		_ = json.NewDecoder(resp.Body).Decode(&res)
+		_ = json.NewDecoder(resp.Body).Decode(res)
```

Comment:
Bitte so lassen. Es braucht die Indirection. Generell eine schlechte Idee, bei PRs ohne Not artfremde Themen zu ändern ;)

Reply:
Hab es reverted.
Aber bitte schau nochmal. Ich bin mir nicht sicher weshalb das im Bestand so ist (und funktioniert?).
Da wird imho viel zu oft hintereinander dereferenziert. Bei XML jedenfalls funktioniert das so nicht.
Aber ich bin jetzt auch nicht so der ultimative Pointer-Freak...

Reply:
Das muss so weil Go immer pass by Value macht. Xml schaue ich nochmal.